README.md

```diff
@@ -20,6 +20,8 @@ Makefile targets are:
 
 This projects has 2 main components; the Radius server, and the healthcheck service.
 
+Its deployments are controlled by a separate app, the [Safe Restarter][safe-restarter]
```

Comment:
Not sure about deployments.
Maybe just say that it is restarted daily by a separate app, the [Safe Restarter][safe-restarter]

Reply:
Good shout, this is more accurate, I spaced.